examples/c++/tca9548a.cxx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Author: Gerard Vidal <gerard.vidal@ens-lyon.fr>
+ * 2017 IFE ENS de Lyon
+ * 
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include <unistd.h>
+#include <iostream>
+#include <signal.h>
+
+//#include "../mcp9808/mcp9808.hpp"
```

Comment:
Could probably remove this?  Even if you wanted to use this sensor, the current method of handling dependencies would handle this for you.  You would add "mcp9808" to the argument list to upm_module_init(), so in your example code you would just use:

`#include "mcp9808.hpp"`

and in your driver CMakeLists.txt file, your call to upm_module init() would look like:

`upm_module_init(mraa mcp9808)`

And the include paths and library dependencies would be handled for you.